packages/nextjs/package.json

```diff
@@ -70,20 +70,20 @@
     "@clerk/backend": "1.0.0-beta.28",
     "@clerk/clerk-react": "5.0.0-beta.30",
     "@clerk/shared": "2.0.0-beta.19",
-    "path-to-regexp": "6.2.1"
+    "path-to-regexp": "6.2.1",
+    "tslib": "2.4.1"
   },
   "devDependencies": {
     "@clerk/types": "4.0.0-beta.20",
     "@types/node": "^18.17.0",
     "@types/react": "*",
     "@types/react-dom": "*",
     "eslint-config-custom": "*",
-    "next": "^13.4.19",
-    "tslib": "2.4.1",
+    "next": "^13.5.4",
     "typescript": "*"
   },
   "peerDependencies": {
-    "next": "^13.0.4 || ^14.0.3",
+    "next": "^13.5.4 || ^14.0.3",
```

Comment:
@nikosdouvlis yes, kind of.
if this is for the changeset entry, i guess it's better to change it to `major`. Since we are in a pre-release tag, it's okay.
I think that customers who want to Clerk with NextJS in production would probably have a newer @13 version with all the security updates.
Wdyt?